lib/puma/dsl.rb

```diff
@@ -810,5 +810,12 @@ def fork_worker(after_requests=1000)
     def nakayoshi_fork(enabled=true)
       @options[:nakayoshi_fork] = enabled
     end
+
+    # The number of requests to attempt inline before sending a client back to
+    # the reactor to be subject to normal ordering.
+    #
+    def max_fast_inline(num_of_requests)
+      @options[:max_fast_inline] = Float(num_of_requests)
```

Comment:
I don't think it really matters, but why a float here? We can't have a non-integer number of requests, can we?

Reply:
To handle the use case mentioned in the issue to be able to set the value to Infinity. Using Float seemed like the simplest solution to me.

Reply:
Got it!